### PR TITLE
Ruby 2.0.0-p598 and Gem update

### PR DESCRIPTION
**Pull request by dirkkelly**
This pull request can supersede https://github.com/codeforamerica/citygram/pull/154, I'll close it accordingly.

Two gems were locked to minors during this update.

```
gem 'georuby', '~> 2.3.0'
gem 'grape', '~> 0.9.0'
```

It is my intention to review all locked gems individually, for now I'd like to be able to ensure we can reliable bundle update on a regular basis.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -39,6 +39,7 @@ class App < Sinatra::Base
 require 'app/models'
 require 'app/routes'
 require 'app/workers'
+require 'active_support'
```
Comment (bigfleet) [inline app.rb]: This is curious-- did one of the gems pull this in, and doesn't any longer?

Comment (dirkkelly) [inline app.rb]: I didn't look into it any further than noticing a jump from 4.1 to 4.2 and solving the exception.

I was surprised it was working pre this as my experience with ActiveSupport is that you have to manually require this.

Would you like me to dig into the cause?

**Comment by bigfleet** [comment]
Am I the only one on earth who never had a problem with a flat 2.0.0 and rbenv?

At any rate, at least some of this work is moot as of f7ec7b7

@dirkkelly If you want to fix some minor versions against master now, I'm happy to accept that PR.

**Comment by dirkkelly** [comment]
Cheers @bigfleet.

**Pull request closed without merging**